etc/cumulus-config/cumulus-config.yml

```diff
@@ -46,16 +46,56 @@ cumulus_tempest_project:
 # stackhpc.os-projects role.
 cumulus_resops_users:
   - name: johng-admin
-    password: johng
+    password:
```

Comment:
Is this a todo? What happens when the password is set to None?

Reply:
its a TODO really, I actually hate the fact this seems to always update the password, and this causes things to error out I believe.